dlrover/python/elastic_agent/monitor/resource.py

```
@@ -170,8 +172,9 @@ def report_resource(self):
                 used_mem,
                 self._gpu_stats,
             )
+            return ""
         except Exception as e:
-            logger.exception(e)
+            return f"{e}"
```

Should raise exception here.

---

Here my design is to catch all exception and handle them to diagnosis agent for further process.